app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :id,
+             :username,
+             :unread_notifications,
+             :unread_personal_messages,
+             :unseen_reviewables,
+             :topic_tracking,
+             :group_inboxes
+
+  def unread_notifications
+    object.all_unread_notifications_count - object.new_personal_messages_notifications_count
+  end
+
+  def unread_personal_messages
+    object.new_personal_messages_notifications_count
+  end
+
+  def unseen_reviewables
+    Reviewable.unseen_reviewable_count(object)
+  end
+
+  def topic_tracking
+    TopicTrackingState.report_totals(object)
+  end
+
+  def include_group_inboxes?
+    scope.user.staff?
+  end
+
+  def group_inboxes
+    group_inbox_data =
+      Notification
+        .unread
+        .where(
+          user_id: scope.user.id,
+          notification_type: Notification.types[:group_message_summary],
+        )
+        .pluck(:data)
+        .to_a
```

Comment:
What is the reason of `to_a` here? pluck should already return an array AFAIK

Reply:
Yup, you're right, it's not needed.